tendrl/commons/event.py

```diff
@@ -0,0 +1,25 @@
+import _socket
+from tendrl.commons.message import Message
+
+
+PATH = "/etc/tendrl/.node_agent_socket"
+
+
+class EventFailed(Exception):
+    pass
+
+
+class Event(object):
+    def __init__(self, message):
+        self.sock = _socket.socket(_socket.AF_UNIX, _socket.SOCK_STREAM)
+        self.write(message)
```

Comment:
@r0h4n when you send the message to socket at that time it should be a json ... so in Event  __init__ you need to pass object, then write method will tack care converting obj into json_str using to_json() in Message class